assign1/index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>📷먀먀먀먀먀 혠찡의 사진관티비 꺅📷</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header id="main_header">먀먀먀먀먀 혠찡의 사진관티비 꺅</header>
```

Comment:
header 는 제목, 로고, 검색 폼, 작성자 이름 등의 요소 를 포함하는 소개 및 탐색에 도움을 주는 콘텐츠를 담는 영역!이기 때문에 정확히는 이 안에서 
```suggestion
      <header><h1>먀먀먀먀먀 혠찡의 사진관티비 꺅</h1></header>
```
이렇게 써주는게 더 시맨틱 하겠죠?!